<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>479</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Moskito Kontroll System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/picture/icon.png</normaloff>:/picture/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="gBox_KOS">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Koordinaten</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string>cm</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-999999999.990000009536743</double>
           </property>
           <property name="maximum">
            <double>999999999.990000009536743</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>X-Koordinaten</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Y-Koordinaten</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Z-Koordinaten</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_z">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string>cm</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-999999999.990000009536743</double>
           </property>
           <property name="maximum">
            <double>999999999.990000009536743</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string>cm</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-999999999.990000009536743</double>
           </property>
           <property name="maximum">
            <double>999999999.990000009536743</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="gBox_man">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Manuell</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Beta</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Alpha</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_alpha">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_beta">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>180</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Btn_konfig">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Moskito konfigurieren</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Btn_manual">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>manuell</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Steuerung</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Koordinaten</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>Manuell</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_control">
           <property name="text">
            <string>Steuerbefehle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Gespeicherte Koordinaten</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBx_Kosave"/>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_KOsave">
           <property name="text">
            <string>Koordinaten speichnern</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="Btn_edit">
             <property name="text">
              <string>ändern</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="Btn_rm">
             <property name="text">
              <string>löschen</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="Btn_saveFile">
             <property name="text">
              <string>in Datei schreiben</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="Btn_readFile">
             <property name="text">
              <string>Datei öffnen</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Optionen</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radBtn_direkt">
           <property name="text">
            <string>Direkt anpeilen</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radBtn_parableN">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Wurfparabel einberechnen </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_standby">
           <property name="text">
            <string>standby</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_laser">
           <property name="text">
            <string>Laser an / aus</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>11</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Btn_aim">
        <property name="text">
         <string>zielen</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_fire">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Zielen &amp; Schießen</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lb_fail">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FAIL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lb_status">
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMoskito">
    <property name="title">
     <string>&amp;Moskito</string>
    </property>
    <widget class="QMenu" name="menuZiehloptionen">
     <property name="title">
      <string>Zieloptionen</string>
     </property>
     <addaction name="act_direct"/>
     <addaction name="act_parableN"/>
    </widget>
    <widget class="QMenu" name="menuKoordinatenspeicher">
     <property name="title">
      <string>Koordinatenspeicher</string>
     </property>
     <addaction name="act_KOsave"/>
     <addaction name="act_edit"/>
     <addaction name="act_rm"/>
     <addaction name="act_readKOFile"/>
     <addaction name="act_saveKOFile"/>
     <addaction name="separator"/>
     <addaction name="act_KOclear"/>
    </widget>
    <addaction name="act_aim"/>
    <addaction name="act_fire"/>
    <addaction name="act_demo"/>
    <addaction name="separator"/>
    <addaction name="act_laser_on_off"/>
    <addaction name="act_standby"/>
    <addaction name="menuZiehloptionen"/>
    <addaction name="menuKoordinatenspeicher"/>
    <addaction name="separator"/>
    <addaction name="act_close"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="act_aboutMoskito"/>
    <addaction name="act_aboutQt"/>
   </widget>
   <widget class="QMenu" name="menuOptionen">
    <property name="title">
     <string>&amp;Optionen</string>
    </property>
    <addaction name="act_manual"/>
    <addaction name="act_rec"/>
    <addaction name="separator"/>
    <addaction name="act_konf"/>
   </widget>
   <widget class="QMenu" name="menuSpezial">
    <property name="title">
     <string>Spezial</string>
    </property>
    <addaction name="act_webPasswd"/>
   </widget>
   <addaction name="menuMoskito"/>
   <addaction name="menuOptionen"/>
   <addaction name="menuSpezial"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="act_aim">
   <property name="text">
    <string>&amp;zielen</string>
   </property>
  </action>
  <action name="act_fire">
   <property name="text">
    <string>&amp;feuern</string>
   </property>
  </action>
  <action name="act_demo">
   <property name="text">
    <string>&amp;Demo</string>
   </property>
  </action>
  <action name="act_KOsave">
   <property name="text">
    <string>neue &amp;Koordinaten speichern</string>
   </property>
  </action>
  <action name="act_aboutMoskito">
   <property name="text">
    <string>Über Moskito</string>
   </property>
  </action>
  <action name="act_aboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="act_standby">
   <property name="text">
    <string>stand&amp;by</string>
   </property>
  </action>
  <action name="act_konf">
   <property name="text">
    <string>Konfigurationsmodus</string>
   </property>
  </action>
  <action name="act_rec">
   <property name="text">
    <string>Steuerbefehle</string>
   </property>
  </action>
  <action name="act_direct">
   <property name="text">
    <string>direkt anpeilen</string>
   </property>
  </action>
  <action name="act_parableN">
   <property name="text">
    <string>Wurfparabel einberechnen</string>
   </property>
  </action>
  <action name="act_close">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="act_manual">
   <property name="text">
    <string>manuelle Bedienung</string>
   </property>
  </action>
  <action name="act_libelle_konf">
   <property name="text">
    <string>Libelle konfiguerien</string>
   </property>
  </action>
  <action name="act_KOclear">
   <property name="text">
    <string>gespeicherte Koordinaten löschen</string>
   </property>
  </action>
  <action name="act_saveKOFile">
   <property name="text">
    <string>in Datei schreiben</string>
   </property>
  </action>
  <action name="act_readKOFile">
   <property name="text">
    <string>aus Datei lesen</string>
   </property>
  </action>
  <action name="act_rm">
   <property name="text">
    <string>löschen</string>
   </property>
  </action>
  <action name="act_edit">
   <property name="text">
    <string>ändern</string>
   </property>
  </action>
  <action name="act_laser_on_off">
   <property name="text">
    <string>Laser on / off</string>
   </property>
  </action>
  <action name="act_webPasswd">
   <property name="text">
    <string>Webpasswort / Webident setzen</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>gBox_KOS</tabstop>
  <tabstop>doubleSpinBox_x</tabstop>
  <tabstop>doubleSpinBox_y</tabstop>
  <tabstop>doubleSpinBox_z</tabstop>
  <tabstop>gBox_man</tabstop>
  <tabstop>spinBox_alpha</tabstop>
  <tabstop>spinBox_beta</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radBtn_direkt</tabstop>
  <tabstop>radBtn_parableN</tabstop>
  <tabstop>Btn_standby</tabstop>
 </tabstops>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Btn_fire</sender>
   <signal>clicked()</signal>
   <receiver>act_fire</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_aim</sender>
   <signal>clicked()</signal>
   <receiver>act_aim</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_KOsave</sender>
   <signal>clicked()</signal>
   <receiver>act_KOsave</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>act_direct</sender>
   <signal>triggered()</signal>
   <receiver>radBtn_direkt</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_standby</sender>
   <signal>clicked()</signal>
   <receiver>act_standby</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>act_parableN</sender>
   <signal>triggered()</signal>
   <receiver>radBtn_parableN</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_konfig</sender>
   <signal>clicked()</signal>
   <receiver>act_konf</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>gBox_KOS</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>gBox_man</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gBox_KOS</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gBox_man</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton_3</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_manual</sender>
   <signal>clicked()</signal>
   <receiver>act_manual</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_readFile</sender>
   <signal>clicked()</signal>
   <receiver>act_readKOFile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_saveFile</sender>
   <signal>clicked()</signal>
   <receiver>act_saveKOFile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_rm</sender>
   <signal>clicked()</signal>
   <receiver>act_rm</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_edit</sender>
   <signal>clicked()</signal>
   <receiver>act_edit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_control</sender>
   <signal>clicked()</signal>
   <receiver>act_rec</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_laser</sender>
   <signal>clicked()</signal>
   <receiver>act_laser_on_off</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
